fairscale/experimental/nn/multiprocess_pipe.py

```diff
@@ -69,6 +70,12 @@ def _rcat(tensors: List) -> Tensor:
     return torch.cat([t.local_value() for t in tensors])
 
 
+def _rcheckpoint(rmodule: rpc.RRef, input_rref: rpc.RRef) -> TensorOrTensors:
+    module = rmodule.local_value()
+    input = module[0](input_rref)  # calls _ToHere.forward
+    return checkpoint_sequential(module[1:], 1, input)
```

Comment:
why not pass input_rref here?

Reply:
checkpoint_sequential expects a Tensor with requires_grad=True. Otherwise, it asserts.